src/pykiso/test_coordinator/test_execution.py

```diff
@@ -433,6 +437,9 @@ def execute(
     except KeyboardInterrupt:
         log.exception("Keyboard Interrupt detected")
         exit_code = ExitCode.ONE_OR_MORE_TESTS_RAISED_UNEXPECTED_EXCEPTION
+    except NoTestsFoundException:
+        log.exception("No tests found in given test suites")
```

Comment:
I don't think that this will prevent the auxiliaries from being created and started at import, but PR #340 will at least ensure that the auxiliaries are properly stopped afterwards